src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/web/RegistrationController.java

```diff
@@ -0,0 +1,45 @@
+package at.ac.tuwien.student.sese2017.xp.hotelmanagement.web;
+
+import at.ac.tuwien.student.sese2017.xp.hotelmanagement.domain.dao.Customer;
+
+import javax.validation.Valid;
+
+import lombok.extern.java.Log;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Log
+@Controller
+public class RegistrationController {

```

Comment:
complete javadocs and explain return values

Reply:
done